molecule/packages-dynamic/molecule.yml

```diff
@@ -43,8 +44,8 @@ provisioner:
       group_vars: ../scenario_resources/group_vars/
 scenario:
   test_sequence:
-    - lint
     - dependency
+    - lint
```

Comment:
yes and no.
No - as we are installing the collection at tox.ini
Yes - if we were using the molecule to install the collection, lint would break because it wouldn't recognize the collection fqdn.
As I started the PR counting on molecule, I did this change, but now we don't need it anymore

Reply:
"Yes and no" sounds like the theme of this PR...
Let's keep it that way, since a user can run this with molecule py3 and ansible 2.10 without the need to install collections by hand.